docs/source/API/core/view/create_mirror.rst

```diff
@@ -15,14 +15,22 @@ A common desired use case is to have a memory allocation in GPU memory and an id
 Usage
 -----
 
+The key difference between ``create_mirror`` and ``create_mirror_view`` is the following: ``create_mirror`` `always` allocates new memory in the specified space (shown below for host space), while ``create_mirror_view`` only allocates memory if the View to be mirrored (``a_view``) is not already accessible from the specified space, and otherwise simply returns ``a_view``.
+Use ``create_mirror_view`` when you want to skip the memory allocation and copy within the same execution space, and use ``create_mirror`` if you need the data to be independent, e.g. for overlapping computation and I/O.
+
 .. code-block:: cpp
 
+    // Both host_mirror and host_mirror_view have the correct properties for deepCopy from/to a_view
+    // host_mirror is guaranteed to have separately allocated memory from a_view
     auto host_mirror = create_mirror(a_view);
+    // host_mirror_view may point to the same memory as a_view, if a_view is host-accessible
     auto host_mirror_view = create_mirror_view(a_view);
 
+    // You can specify the execution space from which the mirror view must be accessible
     auto host_mirror_space = create_mirror(ExecSpace(),a_view);
     auto host_mirror_view_space = create_mirror_view(ExecSpace(),a_view);
```

Comment:
This should really not be called `host_mirror_view_space`. I would rather have an example that provides a memory space instance since there is also the API using `view_alloc` that allows providing an execution space instance that is used for initialization and copying.
```suggestion
    // You can specify the space from which the mirror view must be accessible
    auto mirror = create_mirror(memory_space_instance, a_view);
    auto mirror_view = create_mirror_view(memory_space_instance, a_view);
```